languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -35196,6 +35196,17 @@ USA
             <message>Did you mean <suggestion><match no="1" regexp_match="riel" regexp_replace="rial" /></suggestion> (= substance) or <suggestion><match no="1" regexp_match="eriel" regexp_replace="ériel" /></suggestion> (= military equipment)?</message>
             <example correction="materials|matériels">I needed more electronic <marker>materiels</marker> to build a fully-functional machine.</example>
         </rule>
+        <rule id="BARCA_DIACRITIC" name="Barca or Barça">
```

Comment:
This rule still gives two FPs in the Rule Editor:

- If the latter, then Hannibal and his brothers also bore the name "Barca".
- Barca (, ) was the Semitic surname of his aristocratic family, meaning "shining" or "lightning".

The former can be fixed by adding skip="-1" to the first token. The latter is given away by the word "surname" afterwards. I think this might be a case for [Ruud's trick](https://dev.languagetool.org/tips-and-tricks#warn-for-misused-word-except-in-correct-context).

Reply:
Thanks for the pointer! Does the new antipattern suffice, or is it redundant to except the same keywords in two places?